cmd/restic/cmd_backup.go

```diff
@@ -129,6 +131,9 @@ func init() {
 	f.StringVar(&backupOptions.TimeStamp, "time", "", "`time` of the backup (ex. '2012-11-01 22:08:41') (default: now)")
 	f.BoolVar(&backupOptions.WithAtime, "with-atime", false, "store the atime for all files and directories")
 	f.BoolVar(&backupOptions.IgnoreInode, "ignore-inode", false, "ignore inode number changes when checking for modified files")
+	if runtime.GOOS == "windows" {
+		f.BoolVar(&backupOptions.UseWindowsVSS, "use-windows-vss", false, "use Windows Volume Shadow Copy Service")
```

Comment:
@fgma We've settled on making the option name a bit more generic so that it can be applied to other systems as well in the future.

Would you please be so kind as to change the option name from `--use-windows-vss` to `--use-fs-snapshot` (for "use filesystem snapshot", which if we add similar functionality at some point can then apply to other platforms and filesystems as well)?

My suggestion for the description would then be "use filesystem snapshot where possible (currently only Windows VSS)".

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/dc44ea2b3cab883d6e8d2bf9d357b3cb784f8b0f

I've also enabled the flag for all platforms as the description implies it will only be done if possible and scripts etc. stay cross-platform that way.